airbyte-webapp/src/components/base/Switch/Switch.tsx

```diff
@@ -19,7 +19,7 @@ export const Switch: React.FC<SwitchProps> = ({ loading, small, checked, value,
   });
 
   return (
-    <label onClick={(event: React.SyntheticEvent) => event.stopPropagation()} className={labelStyle}>
```

Comment:
this does not appear to make any change to the behavior of the switch.  tested on connection page  and connections list page manually.

Reply:
++ Even if this is needed for one specific place to work (that we forgot about now), we rather should have that code in place in that specific place and instead expose the `onClick` as a property. 

The only place I can think about maybe worth checking: the enable/disable switch in the connection overview table, that toggling it, does not also jump into the connection (by still triggering the click event of the connection row).